src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -273,6 +273,18 @@ func WithInitContainers(initContainers []corev1.Container) PodOption {
 	}
 }
 
+func WithEmptyInitdata(kbsEndpoint string) PodOption {
```

Comment:
I guess I'm missing something but what about this is empty initdata as you are creating it here?